server/datastore/mysql/queries_test.go

```diff
@@ -474,12 +496,107 @@ func testQueriesList(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 
 	opts := fleet.ListQueryOptions{}
-	results, err := ds.ListQueries(context.Background(), opts)
+	opts.IncludeMetadata = true
+
+	opts.Platform = ptr.String("darwin")
+	// filtered by platform
+	results, count, meta, err := ds.ListQueries(context.Background(), opts)
+	require.NoError(t, err)
+	require.Equal(t, 2, len(results))
```

Comment:
Double checking it should match queries from 4 to 9 because empty platform means run on all platforms.